datus/agent/node/generate_sql_node.py

```
@@ -200,3 +233,74 @@ async def _generate_sql_stream(
+            sql_query = sql_query.strip().replace("```json\n", "").replace("\n```", "")
+            # Remove SQL comments
+            cleaned_lines = []
+            for line in sql_query.split("\n"):
+                line = line.strip()
+                if line and not line.startswith("--"):
+                    cleaned_lines.append(line)
+            cleaned_sql = " ".join(cleaned_lines)
+            try:
+                sql_query_dict = json.loads(cleaned_sql)
+            except json.JSONDecodeError:
+                logger.error(f"Failed to parse cleaned SQL: {cleaned_sql}")
```

Ruff (0.14.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)